### ISSUE TITLE
Make correct color automatically selected

### ISSUE THREAD

**Issue by liambloom** [liambloom/my-site#7]
When the page is sent to the client, the radio button group for theme should have the default theme (or whatever theme the page is served with) already checked.

**Comment by liambloom** [comment]
One thing may make managing the theme easier is if it were an enum:

```rust
#[derive(Debug, Copy, Clone)]
enum Theme {
  Light,
  Dark,
}

impl fmt::Display for Theme {
  fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
    match self {
      Theme::Light => write!(f, "light"),
      Theme::Dark => write!(f, "dark"),
    }
  }
}
```

**Comment by liambloom** [comment]
Fixed in commit 117f4a55058801bd2d45956fbfc7d870a5e7de57.
